index.html

```diff
@@ -0,0 +1,72 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Welcome - I'am Emmanuel Ogah, a software developer</title>
+    <link rel="stylesheet" href="css/styles.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,100;0,200;0,400;0,700;1,100;1,300&display=swap" rel="stylesheet">
+</head>
+<body>
+    <div class="nav-container">
+        <div class="wrapper">
+            <div  class="logo">
+                <img src="images/mobile/my-logo.png" alt="image"/>
```

Comment:
- [ ]  Your logo should be wrapped in an `a` tag as it is supposed to be a link.

Reply:
The logo is now wrapped in a tag